### DISCUSSION TITLE
Feel much worse about Copilot Chat these days

### DISCUSSION THREAD

**Discussion by LikeDreamwalker** [community/community#63456] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I can't tell why but Copilot Chat is worse to use these days for me. Now Chat will focus on the information I gave her earlier, but it always runs in a dead loop.
I don't have a typical example, but I can describe it. Like I want a function about A, which will achieve goal B, and in a way like C.
Weeks ago, when way C was not helpful, Chat would try way D, way E, way F, and so on, but now it is just running in way C1, way C2, and way C3, and can't give me different advice. And this makes using Chat become un-inspired. What's more, if I use Copilot to code inline, I will be much more efficient than Chat.
Is there anyone who also has this feeling? And should I change the way to talk to Chat?

**Comment by Akash1134** [comment]
Hi @LikeDreamwalker ,

Depending on your scenario and input data, you could experience different levels of performance. While not limited specifically, GitHub Copilot Chat beta has [limitations](https://docs.github.com/en/en/copilot/github-copilot-chat/about-github-copilot-chat#limitations-of-github-copilot-chat), which may include:


- Potential inaccuracies and irrelevant suggestions
- Lack of context awareness beyond the provided code context
- Lack of deep domain knowledge or expertise in niche programming languages
- Biases present in the training data that can influence the suggestions
- The code suggestions can be rendered vulnerable due to vulnerabilities present in the training data

Users should be cautious of security and privacy concerns and avoid sharing sensitive information. It is important to exercise judgment, review suggestions, and ensure the generated code aligns with specific requirements and best practice.

Thanks!
